src/EnergyPlus/EnergyPlus.hh

```diff
@@ -91,7 +91,6 @@ typedef std::int64_t Int64;
 #include <ObjexxFCL/Array5D.fwd.hh>
 #include <ObjexxFCL/MArray1.fwd.hh>
 #include <ObjexxFCL/Omit.hh>
-#include <ObjexxFCL/Optional.fwd.hh>
```

Comment:
Removing this means no more usage in interfaces for free just by including `EnergyPlus.hh`. The intent here is to encourage usage of Optional to be accompanied by an inclusion of an Optional header, which kind of works.

Reply:
:+1: